interfaces/Portalicious/src/app/components/page-layout/components/registration-header/add-note-form/add-note-form.component.html

```diff
@@ -0,0 +1,41 @@
+<app-form-sidebar
+  [(visible)]="formVisible"
+  [formGroup]="formGroup"
+  [mutation]="addNoteMutation"
+  formTitle="Add note"
+  i18n-formTitle="@@add-note"
+>
+  <p
+    i18n
+    class="font-semibold"
+  >
+    You are about to add a note to {{ registrationName() }}'s profile.
+  </p>
+  <p
+    i18n
+    class="mb-8"
+  >
+    The note will be visible to other 121 users who are permitted to view
+    personal profiles.
+  </p>
+  <app-form-field-wrapper
+    label="Note"
+    i18n-label
+    [errorMessage]="formFieldErrors()('note')"
+  >
+    <textarea
+      rows="5"
+      pInputTextarea
+      formControlName="note"
+      autocomplete="off"
+    ></textarea>
+    <ng-container slot="details">
+      <p
+        i18n
+        class="text-grey-500 txt-system-s"
+      >
+        Tip: Press Command / Ctrl + Enter to submit.
```

Comment:
Oe. missed opportunity to use `<kbd>`-elements! :D

Reply:
🫨  I was today years old when I learned.... I am making this change now :D

Reply:
Nevermind - I tried it and I didn't like it 😄

Reply:
Yeah, I wondered what the default/PrimeNG styling would be.. ;)
But the "semantic"-benefits are negligible, I think...

Maybe, some day, when we add a `⌘+K` power-users keyboard-shortcuts-panel, etc.. ;)